catalogue/webapp/pages/search/images.tsx

```diff
@@ -216,6 +217,9 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
         }),
       };
     }
+    if (params.color) {
+      params.color = params.color.replace('#', '');
```

Comment:
Why do we do this again? I think you explained to me at the office...

Reply:
I'll add a comment to explain moving forward